cmd/generate.go

```diff
@@ -101,6 +108,8 @@ function create_west_cluster() {
   roachprod create "$WEST_CLUSTER" -u $USER -n $NODES --lifetime "6h" --clouds "$CLOUD" \
     --$CLOUD-machine-type "{{.MachineType}}" {{.NodeWestLocation}} {{.EvaledArgs}} {{.UsWestAmi}} \
     --label {{.Usage}}
+
+  roachprod run "$WEST_CLUSTER":1 -- cat /proc/cpuinfo | grep "model name" | sort | uniq -c | sort -rn > "$logdir"/"$WEST_CLUSTER"_cpu.txt
```

Comment:
Wouldn't each vcore be of the exactly same model? Also, shouldn't we just dump the full contents of `cpuinfo` as in the above?

Reply:
Yep just dump the info of the first node to the log file now. Thanks!